lib/Rx/Observable/BaseObservable.php

```diff
@@ -311,8 +330,14 @@ public function groupByUntil($keySelector, $elementSelector = null, $durationSel
             throw new InvalidArgumentException('Element selector should be a callable.');
         }
 
+        if (null === $durationSelector) {
+            $durationSelector = function($x) { return $x; };
+        } else if ( ! is_callable($durationSelector)) {
+            throw new InvalidArgumentException('Duration selector should be a callable.');
```

Comment:
Can you add a regression test for this? E.g. https://github.com/asm89/Rx.PHP/blob/c923ad7b9430b4a4197d1f93c7d637e19cf3e1ca/test/Rx/Functional/Operator/GroupByTest.php#L297.